backend/python/app/services/implementations/story_service.py

```diff
@@ -894,6 +897,30 @@ def get_story_translation_statistics(self):
             num_translations_completed=num_translations_completed,
         )
 
+    def soft_delete_story(self, id):
+        try:
+            story = StoryAll.query.get(id)
+            for content in story.contents:
+                content.is_deleted = True
+
+            story.is_deleted = True
+            story.translated_languages = None
+
+            # Remove story_translations for this story
```

Comment:
use the service function soft_delete_story_translation on all the story_translations' ids. Soft deleting a story translation involves more than just setting the is_delete col on the story translation